### PR TITLE
Update demo to use 0.17.10 Android SDK

### PR DESCRIPTION
**Pull request by linsang21**
*Issue #, if available:*
N/A

*Description of changes:*
Upgrade Android SDK version to 0.17.10, fix build error. Tested joining meeting with web, send / receive audio / video.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -61,5 +61,20 @@ allprojects {
         flatDir {
             dirs 'libs'
         }
+
+        exclusiveContent {
```
Comment (hokyungh) [inline android/build.gradle:65]: why is this needed?

Comment (linsang21) [reply android/build.gradle:65]: When I build the app, I got 
```
* What went wrong:
Execution failed for task ':app:mergeDebugNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libfbjni.so' from inputs:
      - /Users/linsang/.gradle/caches/transforms-3/4f41139e8a8b9fc7536f2fec86e6b63a/transformed/jetified-react-native-0.71.0-rc.0-debug/jni/arm64-v8a/libfbjni.so
      - /Users/linsang/.gradle/caches/transforms-3/84995423265cc7a30d31c99b7568043a/transformed/jetified-fbjni-0.3.0/jni/arm64-v8a/libfbjni.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets
```

Comment (hokyungh) [reply android/build.gradle:65]: Do you have same issue if you use [`mavenCentral()`](https://github.com/aws-samples/amazon-chime-react-native-demo/pull/173/files#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR58)?

Comment (linsang21) [reply android/build.gradle:65]: What do you mean? Thought we had mavenCentral defined in L49?

**Comment by linsang21** [comment]
Discard in favor of https://github.com/aws-samples/amazon-chime-react-native-demo/pull/173
